src/components/ProgressIndicator/readme.md

```diff
@@ -438,3 +438,89 @@ class OnClickProgressIndicator extends React.Component {
 
     <OnClickProgressIndicator />;
 ```
+
+##### numeric variant
+
+```js
+import React, { useState } from 'react';
+import { ProgressIndicator, ProgressStep, ButtonGroup, ButtonIcon } from 'react-rainbow-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlus, faEllipsisV } from '@fortawesome/free-solid-svg-icons';
+
+const NumericProgressIndicator = () => {
+    const [currentStepName, setCurrentStepName] = useState('step-1');
+
+    const handleOnClick = (event, name) => setCurrentStepName(name);
+
+    return (
+        <div className="rainbow-m-bottom_large rainbow-p-bottom_large">
+            <GlobalHeader
+                src="images/user/user3.jpg"
+                className="rainbow-p-bottom_medium rainbow-m-bottom_medium"
+            >
+                <ButtonGroup>
+                    <ButtonIcon
+                        icon={<FontAwesomeIcon icon={faPlus} />}
+                        variant="border-filled"
+                        disabled
+                    />
+                    <ButtonIcon
+                        icon={<FontAwesomeIcon icon={faEllipsisV} />}
+                        variant="border-filled"
+                        disabled
+                    />
+                </ButtonGroup>
+            </GlobalHeader>
```

Comment:
@TahimiLeonBravo @LeandroTorresSicilia @maxxgreene I believe that in all the ProgressIndicator examples the GlobalHeader block does not fulfill any objective, it only complicates the example.

Reply:
Agreed, I consulted with @TahimiLeonBravo and she said that it could be removed, so i did.